Makefile

```diff
@@ -91,7 +91,7 @@ check-examples: valadoc-example-tester
 
 build-data:
 	npm install
-	./node_modules/.bin/gulp
+	npx gulp
```

Comment:
This would require downloading gulp every time you run `make serve-mini`, and gulp is already installed. I run `make serve-mini` for every code change - this will make things really slow.

Reply:
`npx` will use the local installed package, global installed package, or download from npm, in that order.

Reply:
alright, that makes it not slow, but why have `npx` here anyway?

Reply:
It's more "technically" correct then the full path in node modules. If for some reason `npm` changes where the bin files are or if the binary gets renamed `npx` will still work.

:man_shrugging: It's an optional change